databricks/koalas/plot.py

```diff
@@ -90,8 +90,9 @@ def get_sampled(self, data):
             sampled = data._sdf.sample(fraction=float(self.fraction))
             return DataFrame(data._internal.copy(sdf=sampled)).to_pandas()
         elif isinstance(data, Series):
+            scol = data._kdf._internal.data_scols[0]
```

Comment:
oops, it should be `data._scol` so that it respects the expression stored in Series. Let me fix it quick.

Reply:
oops... i didn't see this option in `Series.py`... my bad, sorry for this, i see it is not fixed yet, i will just create a small PR to fix it quick and validate if plot still works. @HyukjinKwon

Reply:
ah, I just made a one line fix just now - I found this comment after opening the PR :D. It's fine!

Reply:
😅 oh, sorry, i waked up and saw the comment, and found it hadn't been fixed yet in master branch, i thought you might forget this issue, so i opened a small PR for this ... but thanks for the tip, appreciate it!